src/utils/foundry.ts

```diff
@@ -246,7 +247,20 @@ const watchForgeOutput = (
   const magicEmojis = ['🧙', '🪄', '🧚', '✨'];
 
   stdout.on('data', (chunk: any) => {
-    // if the chunk contains the succes label, then it's related to a transaction
+    if (state.disableLogging) {
+      if (!chunk.toString().includes('contracts were verified!')) return;
+      // resume logging
+      else state.disableLogging = false;
+    }
```

Comment:
❓ What happens when contracts fail to verify and `contracts were verified!` never appears? Does the UX make sense if logging is disabled and is never re-enabled?

Reply:
the verification request always returns a success message